index.js

```diff
@@ -199,6 +211,10 @@ app.on('ready', () => {
       }
     });
 
+    win.on('focus', () => {
+      win.focusTime = process.uptime();
```

Comment:
I decided to use this approach because it's easier and cleaner in my opinion: to handle windows closed and removed, we should store not only the last focued win, but also the previous ones to fallback in case of window closing. Doing this with process.uptime() way allow us to not have to handle window closing creating complex logic to order, delete and update a _focusedWindows_ array.